posthog/urls.py

```diff
@@ -106,6 +106,7 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> UR
     opt_slash_path("batch", capture.get_event),
     opt_slash_path("s", capture.get_event),  # session recordings
     opt_slash_path("robots.txt", robots_txt),
+    opt_slash_path("security.txt", security_txt),
```

Comment:
As per proposed [spec](https://securitytxt.org/) I think it should be placed at `.well-known/security.txt`